TODO/world-class-testing-development-pipeline-for-android.md

```diff
@@ -1,64 +1,63 @@
 * 原文链接 : [World-Class Testing Development Pipeline for Android - Part 1.](http://blog.karumi.com/world-class-testing-development-pipeline-for-android/)
 * 原文作者 : [Karumi](hello@karumi.com)
 * 译文出自 : [掘金翻译计划](https://github.com/xitu/gold-miner)
-* 译者 : [认领地址](https://github.com/xitu/gold-miner/issues/111)
-* 校对者: 
-* 状态 : 认领中
+* 译者 : [markzhai](https://github.com/markzhai)
+* 校对者:
+* 状态 : 翻译完成
 
-# 世界级的 Android 测试流程（一）
+# 世界级的 Android 测试开发流程（一）
 
-After developing mobile applications and collaborating with manual QA teams for several years we decided to start writing tests. We knew that, as engineers, **test automation is the key to successful mobile development.** In this blog post we’d like to share how our story, the Karumi testing story, started several years ago. This is the first blog post of a series where we will cover all aspects of world-class testing development pipeline for Android.
+在开发完移动应用并和手动QA团队合作了数年后，我们决定开始写测试。作为工程师，我们知道，**自动化测试是成功的移动开发之关键。** 在这篇博客里，将会分享我们的故事，Karumi测试故事, 开始于几年前。这是系列博客的第一篇，我们将会囊括世界级的 Android测试流程的所有方面。
 
-A couple of years ago we started writing tests for mobile applications. We had limited testing knowledge so we focused on acceptance and unit tests using the most common frameworks, a simple test runner and a mocking library. After some time we ran into problems:
+几年前，我们开始为移动应用写测试。我们对测试了解有限，所以我们致力于接受测试并使用最常用的框架来做单元测试，一个简单的test runner和mocking库。过了一段时间我们遇到了问题：
 
-- We had no idea what to test and how to test it.
-- Our code was not ready to be tested.
-- We were obsessed with Mike Cohn’s Test Pyramid without thinking about the type of software we were writing.
-- Just because our tests were passing didn’t mean the code was working.
+- 我们不知道测试什么，又如何去测试它。
+- 我们的代码还没准备好被测试。
+- 我们沉迷于Mike Cohn的测试金字塔，却没有考虑到我们在写的软件类型。
+- 我们的测试通过了也不意味着代码就没问题。
 
```

Comment:
We had no idea what to test and how to test it.
我们不知道测试什么和如何去测试它
Just because our tests were passing didn’t mean the code was working.
即便我们的测试通过了，也不意味着代价没有问题。